### PR TITLE
Use Armadillo Service's Admin and Storage APIs

### LINKED ISSUE #28: list_folders fails if no folders found
```
> library(MolgenisArmadillo)
> armadillo.list_folders()
Error in startsWith(bucket_names, prefix) : non-character object(s)
Called from: startsWith(bucket_names, prefix)
```

### LINKED ISSUE #49: When creating a duplicate project you get the stacktrace, this needs to be converted to something like "Project already exists"

### PR DESCRIPTION
**Pull request by tommydeboer**
Closes #49 
Closes #28 

- No longer interacts with MinIO directly
- Uses new Admin and Storage APIs

### CODE REVIEW

**Review by mswertz — COMMENTED** [review]
Code looks good. Just want to run it locally.

I noticed there is no docs on the 'devtools' but I found out how to do it.

File: DESCRIPTION [hunk 1]
```diff
@@ -37,10 +37,7 @@ Depends:
     R (>= 3.6)
 Imports:
     httr,
-    urltools,
     xml2,
-    aws.iam,
-    aws.s3,
     MolgenisAuth,
```
[the remark this answers is not in the document]
Comment (tommydeboer) [reply DESCRIPTION:41]: MolgenisAuth is our R authentication library. Both the Armadillo client and the Datashield needed it so we extracted and published it separately.

[the remark this answers is not in the document]
Comment (tommydeboer) [reply DESCRIPTION:41]: See https://github.com/molgenis/molgenis-r-auth

**Comment by tommydeboer** [comment]
@mswertz 
>I noticed there is no docs on the 'devtools' but I found out how to do it.

Do you mean how to install? We can add that to the readme

**Comment by tommydeboer** [comment]
[Adapt the armadillo client to use armadillo service instead of minio](https://trello.com/c/aLzZZ9em/928-adapt-the-armadillo-client-to-use-armadillo-service-instead-of-minio)

**Comment by mswertz** [comment]
> Do you mean how to install? We can add that to the readme

Also how to develop and test (which is maybe what you meant). So when somebody checks out the code (s)he knows how to run in development. Just main commands plus I suppose hyperlink to some general manuals on packages (where I found what to do).

**Review by mswertz — APPROVED** [review]

**Pull request merged**